mobile/src/main/java/com/alexstyl/specialdates/util/Notifier.java

```diff
@@ -89,19 +93,53 @@ public void forDailyReminder(ContactEvents events) {
                 );
 
         String title = NaturalLanguageUtils.joinContacts(context, events.getContacts(), 3);
-        String fullText = TextUtils.join(", ", events.getContacts());
 
         NotificationCompat.Builder builder = new NotificationCompat.Builder(context)
                 .setSmallIcon(R.drawable.ic_stat_contact_event)
                 .setContentTitle(title)
                 .setLargeIcon(largeIcon)
-                .setStyle(new NotificationCompat.BigTextStyle().bigText(fullText))
                 .setVisibility(NotificationCompat.VISIBILITY_PRIVATE)
                 .setAutoCancel(true)
-                .setContentText(fullText)
                 .setContentIntent(intent)
+                .setNumber(events.getContactCount())
                 .setColor(context.getResources().getColor(R.color.main_red));
 
+        // if event.getContacts() > 1 then use InboxStyle, otherwise the BigTextStyle like here:
+        if (events.getContacts().size() == 1) {
+            int age = events.getContacts().get(0).getBirthday().getAgeOnYear(events.getDate().getYear());
```

Comment:
Also, make sure to get the contact out of `events.getEvent(i)`.
ContactEvents#getContacts() does not map the events inside the class. You can have multiple events for one contact.